includes/embeds/class-amp-scribd-embed-handler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Class AMP_SoundCloud_Embed_Handler
+ *
+ * @package AMP
+ * @since 1.3.1
+ */
+
+/**
+ * Class AMP_Scribd_Embed_Handler
+ */
+class AMP_Scribd_Embed_Handler extends AMP_Base_Embed_Handler {
+
+	/**
+	 * Registers embed.
+	 */
+	public function register_embed() {
+		add_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10, 2 );
+	}
+
+	/**
+	 * Unregisters embed.
+	 */
+	public function unregister_embed() {
+		remove_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10 );
+	}
+
+	/**
+	 * Filter oEmbed HTML for Scribd to to be AMP compatible.
+	 *
+	 * @param string $cache Cache for oEmbed.
+	 * @param string $url   Embed URL.
+	 * @return string Embed.
+	 */
+	public function filter_embed_oembed_html( $cache, $url ) {
+		if ( false === strpos( wp_parse_url( $url, PHP_URL_HOST ), 'scribd.com' ) ) {
```

Comment:
This is not 100% foolproof, as it only checks for a partial domain match. An embed for something like `https://unscribd.com` would match as well.

It would be better to check for exact domains to make be safe.
```suggestion
		if ( ! in_array( wp_parse_url( $url, PHP_URL_HOST ), [ 'scribd.com', 'www.scribd.com' ], true ) ) {
```